src/community/script/core/src/main/java/org/geoserver/script/rest/service/SessionService.java

```diff
@@ -0,0 +1,143 @@
+/** (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ *
+ * @author David Vick, Boundless 2017
+ **/
+package org.geoserver.script.rest.service;
+
+
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+import org.geoserver.rest.RequestInfo;
+import org.geoserver.rest.RestException;
+import org.geoserver.script.ScriptManager;
+import org.geoserver.script.ScriptSession;
+import org.geoserver.util.IOUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.Writer;
+import java.net.URI;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Service
+public class SessionService {
+    static Logger LOGGER = Logger.getLogger("org.geoserver.script.rest");
+
+    @Autowired
+    ScriptManager scriptMgr;
+
+    /**
+     * Get a list of Scripting Sessions
+     * @return
+     */
```

Comment:
This javadoc is kind of useless; either improve it or remove it. Perhaps just add a top-level description to the class?

Reply:
removed.

Reply:
fixed.